src/commandline.cpp

```diff
@@ -187,6 +187,8 @@ std::optional<int> CommandLine::process(const std::wstring& line)
         // not a shortcut, try a link
         if (isNxmLink(qs)) {
           m_nxmLink = qs;
+        } else if (qs.startsWith("https://", Qt::CaseInsensitive) || qs.startsWith("http://", Qt::CaseInsensitive)) {
```

Comment:
I don't think `http://` download links should be allowed.

Reply:
It is about what `user` supplies to MO2. Links posted on the internet may begin with `http:` (this happens), when in fact, the lkink would be immediately redirected to `https`. If you want user to see a warning message, manually fix the link and run again, sure, I can add that.

Reply:
I understand why you would do that but that's a very bad idea, especially knowing what's possible to do with mods. If people still post `http://` links on their website, then that's the website faults, MO2 should not have to deal with that, we're in 2023.

I would simply not handle the link. Users are unlikely to run this command manually, so the warning would only be seen by the external tools that would dispatch link, in which case the check could be made there and MO2 would just have to return an error code.